test/jdk/sun/security/lib/cacerts/VerifyCACerts.java

```diff
@@ -269,6 +269,10 @@
             add("addtrustexternalca [jdk]");
             // Valid until: Sat May 30 10:44:50 GMT 2020
             add("addtrustqualifiedca [jdk]");
+            // Valid until: Fri Jan 01 15:59:59 PST 2021
+            add("verisigntsaca [jdk]");
+            // Valid until: Fri Jan 01 15:59:59 PST 2021
+            add("thawtepremiumserverca [jdk]");
```

Comment:
Good. When should we remove outdated ones?

Reply:
I think they'll remove that stuff in January